src/main/java/ch/heigvd/amt/mvcProject/application/question/QuestionCommand.java

```diff
@@ -15,13 +17,15 @@
 @EqualsAndHashCode
 public class QuestionCommand {
```

Comment:
pas d'id pour les questions ?

Reply:
C'est une Command, you noob, on va pas commencer à mettre des Id auto-généré dans la Command